pkg/cmd/codespace/create_test.go

```diff
@@ -490,16 +490,6 @@ func TestBuildDisplayName(t *testing.T) {
 		prebuildAvailability string
 		expectedDisplayName  string
 	}{
-		{
-			name:                 "prebuild availability is pool",
-			prebuildAvailability: "pool",
-			expectedDisplayName:  "4 cores, 8 GB RAM, 32 GB storage (Prebuild ready)",
-		},
-		{
-			name:                 "prebuild availability is blob",
-			prebuildAvailability: "blob",
-			expectedDisplayName:  "4 cores, 8 GB RAM, 32 GB storage (Prebuild ready)",
-		},
```

Comment:
Can we add a test for the new `ready` case you've introduced?

Reply:
@josebalius This doesn't add a new case? It just deletes blob and pool.